libs/eo/landing-page/shell/src/lib/eo-landing-page-shell.component.ts

```diff
@@ -35,17 +36,336 @@ const selector = 'eo-landing-page-shell';
+
+          background: var(--watt-color-primary); // [1]
+          color: var(--watt-color-primary-contrast); // [1]
+
+          min-width: 6.25rem; // [2]
+          height: var(--height); // [3]
+          padding: var(--inset-squish-m--y) var(--inset-squish-m--x); // [3]
+
+          line-height: calc(
+            var(--height) - 2 * var(--inset-squish-m--y)
+          ); // [3] [4]
+
+          &:hover {
+            text-decoration: none; // [1]
+          }
+        }
+
+        .font-primary-color {
+          color: var(--watt-color-primary);
+        }
+
+        .${selector}__p {
+          @include watt.typography-watt-text-m; // This overrides the styles applied from Angular Material on p tags
+        }
+
+        .${selector}__h1 {
+          @include watt.typography-watt-headline-1; // This overrides the styles applied from Angular Material on h1 tags
+        }
+
+        .${selector}__h2 {
+          @include watt.typography-watt-headline-2; // This overrides the styles applied from Angular Material on h2 tags
+          text-transform: none; // This overrides the uppercased styling from watt
+        }
+
+        .${selector}__img {
+          width: 100%;
+          height: auto;
+        }
+
+        .${selector}__footer-mesh {
+          width: 35%;
+          position: absolute;
+          bottom: 200px; // Position the image just above the footer
+        }
+
+        .${selector}__header-mesh {
+          width: 35%;
+          position: absolute;
+          top: 64px; // Position the image just below the header
+          right: 0;
+        }
+
+        .${selector}__development-notification {
+          background: rgba(223, 235, 218, 0.9);
+          top: -40px;
+          width: 100%;
+          height: 80px;
+          position: absolute;
+
+          > p {
+            margin-left: 75px;
+          }
+
+          &::before {
+            display: block;
+            content: ' i ';
+            width: 50px;
+            height: 50px;
+            border-radius: 50%;
+            background: var(--watt-color-primary-dark);
+            color: #fff;
+            position: absolute;
+            left: 16px;
+            text-align: center;
+            line-height: 50px;
+            font-size: 30px;
+          }
+        }
       }
     `,
   ],
   template: `
     <eo-landing-page-header></eo-landing-page-header>
+
+    <img
+      src="/assets/landing-page-mesh-top.png"
+      class="${selector}__header-mesh"
+    />
+
+    <div class="${selector}__wrapper">
+      <div
+        class="${selector}__content ${selector}__display-flex eo-padding-top-xl eo-padding-bottom-xl"
+      >
+        <div class="${selector}__column--large">
+          <h1 class="${selector}__h1">
+            Access
+            <span class="font-primary-color"
+              >your emissions and<br />energy origin</span
+            >
+            overview
+          </h1>
+          <div style="height: 403px; background: #CCC;">
```

Comment:
nitpick: Prefer lower case hex alpha characters.
```suggestion
          <div style="height: 403px; background: #ccc;">
```

Reply:
question: Do we have a CSS variable for this color?

Reply:
Nope - It is a placeholder until Kenneth have the graphics ready only